en/docs/setup/uninstall.md

```diff
@@ -26,15 +50,50 @@ To completely remove APK from your Kubernetes cluster, follow the steps given be
 
 3. Then delete the CRDs using the following command.
 
-    ```
-    kubectl delete -f crds.yaml
-    ```
+```
+kubectl delete -f crds.yaml
+```
 
-4. You will then have to delete the validating and mutating webhook configurations using the following command.
+1. You will then have to delete the validating and mutating webhook configurations using the following command.
```

Comment:
It seems there's a discrepancy in the numbering of the steps. Line 51 has the 3rd step and this step was changed to 1. Please check this.

Reply:
Fixed in https://github.com/wso2/docs-apk/pull/583